UI/WebRTC/WebRTC/WebRTC.Shared/App.xaml.cs

```diff
@@ -171,8 +171,6 @@ private static void InitializeLogging()
 				// Debug JS interop
 				// builder.AddFilter("Uno.Foundation.WebAssemblyRuntime", LogLevel.Debug );
 			});
-
-			global::Uno.Extensions.LogExtensionPoint.AmbientLoggerFactory = factory;
```

Comment:
Why did you need to remove it? This is in the original template, and we may need to adjust that piece.

Reply:
I back up the code and did the required adjustments.